packages/@aws-cdk/aws-ecr-assets/test/dockerignore-image-advanced/.dockerignore

```diff
@@ -0,0 +1,22 @@
+# This a comment, followed by an empty line
```

Comment:
Can we do these tests in memory?

I'm thinking of a test where we have preloaded this list of ignore patterns, and pushing a number of paths past it (either by means of `shouldExclude()` or by means of a new `IgnoreList` API)

That is effectively the same and easier to change/assert against than making a whole new fixture with "just these" files in it.

Reply:
I ended up creating a `createFsStructureFromTree` function, and removing the previous fixtures (with the exception of `demo-image/`).

Does that work for you?